orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/backup.sls

```diff
@@ -9,7 +17,11 @@ include:
 {% if 'None' != configure_remote_db %}
 backup_postgresql_db:
   cmd.run:
-    - name: /opt/salt/scripts/backup_db.sh {{salt['pillar.get']('disaster_recovery:object_storage_url')}} {{salt['pillar.get']('postgres:clouderamanager:remote_db_url')}} {{salt['pillar.get']('postgres:clouderamanager:remote_db_port')}} {{salt['pillar.get']('postgres:clouderamanager:remote_admin')}} {{salt['pillar.get']('disaster_recovery:ranger_admin_group')}} {{salt['pillar.get']('disaster_recovery:close_connections')}} {{salt['pillar.get']('disaster_recovery:database_name') or ''}}
+  {% if '0' == compression_level %}
+    - name: /opt/salt/scripts/backup_db.sh {{object_storage_url}} {{remote_db_url}} {{remote_db_port}} {{remote_admin}} {{ranger_admin_group}} {{close_connections}} {{database_name}}
+  {%- else %}
+    - name: /opt/salt/scripts/backup_db_compressed.sh -s {{object_storage_url}} -h {{remote_db_url}} -p {{remote_db_port}} -u {{remote_admin}} -r {{ranger_admin_group}} -c {{close_connections}} -z {{compression_level}} {{database_name}}
+  {% endif %}
     - require:
```

Comment:
You could invoke backup_db_compressed.sh all the time.

Reply:
done